viewing_party/party.py

```diff
@@ -2,22 +2,178 @@
 
 def create_movie(title, genre, rating):
     pass
+    #make a dictionary with these keys: "title","genre","rating"
+    movie_dict = {}
+    movie_dict["title"]= title
+    movie_dict["genre"] = genre
+    movie_dict["rating"] = rating

```

Comment:
this works just fine! We could also do this all at once:

```suggestion
    movie_dict = {
        movie_dict["title"]: title,
        movie_dict["genre"]: genre,
        movie_dict["rating"]: rating
    }
```
Now, obviously this isn't one line like this, so we aren't saving space. But we can do it all in one expression instead of 4.